src/tzdata/TZData.jl

```diff
@@ -2,6 +2,14 @@ module TZData
 
 using Printf
 using ...TimeZones: DEPS_DIR
+if VERSION >= v"1.4"
+    using Pkg.Artifacts
```

Comment:
Pkg.Artifacts is actually available in v1.3

Reply:
Yes, but because of https://github.com/JuliaLang/Pkg.jl/issues/1912 it's not working with variable artifacts, so I don't see the point in using the module if I can't actually install the artifacts in this version because of this bug.

Reply:
Ah, I see.